config/samples/cloud_resource_config.yaml

```diff
@@ -3,7 +3,9 @@ apiVersion: v1
 metadata:
   name: cloud-resource-config
 data:
-  managed: |
+  openshift: |
+    {"blobstorage":"openshift", "redis":"openshift", "postgres":"openshift"}
+  aws: |
     {"blobstorage":"aws","redis":"aws", "postgres":"aws"}
-  workshop: |
```

Comment:
workshop is for installing in cluster. I don't think we need to remove it?

Reply:
`workshop` isn't exactly removed. I made modification to the naming.
There used to be two deployment types: `workshop` and `managed`.
Since we're introducing GCP support, these names don't make sense anymore.
The deployment types are now:
- `workshop` -> `openshift`
- `managed` -> `aws`
- `gcp` - new type
This way the deployment types will match the provider names and reduce confusion.